src/components/shared/forms/editUserForm.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { useState } from 'react'
+
+import { User, ExpressValidationError } from 'devu-shared-modules'
+
+import { useAppDispatch } from 'redux/hooks'
+import { SET_ALERT } from 'redux/types/active.types'
+
+import TextField from 'components/shared/inputs/textField'
+
+import RequestService from 'services/request.service'
+
+type Props = {
+  user: User
+}
+
+const EditUserForm = ({ user }: Props) => {
+  const [formData, setFormData] = useState<User>(user)
+  const setAlert = useAppDispatch(SET_ALERT)
+
+  const handleUpdatePreferredName = (preferredName: string) => setFormData({ ...formData, preferredName })
+
+  const handleSubmit = () => {
+    RequestService.put(`/api/users/${user.id}`, formData)
+      .then(() => {
+        setAlert({ autoDelete: false, type: 'success', message: 'User Preferences Updated' })
```

Comment:
You don't need to wrap this in brackets, imo it's a bit cleaner not being wrapped.

Reply:
Also I think this would probably be better as an autoDeleted message